content/en/references/changelog.md

```diff
@@ -10,13 +10,30 @@ cascade:
 
 ## Introduction
 
-This page documents the release notes for official LocalStack major and minor releases since LocalStack v1.0.0.  If you're looking for information about nightly releases, beta features, or experimental features, pull the [latest Docker image]({{< ref "docker-images" >}}). The changelog is updated with every release. Updates that affect only LocalStack Web Application or features in beta or limited release may not be reflected.
+This page documents the release notes for official LocalStack major and minor releases since LocalStack v1.0.0.
+If you are looking for information about nightly releases, preview features, or experimental features, pull the [latest Docker image]({{< ref "docker-images" >}}).
+The changelog is updated with every release.
+Updates that affect only LocalStack Web Application or features in preview or limited release may not be reflected.
+
+## Features under Development
+
+LocalStack uses the following terminology to communicate features under development:
+
+* **Preview** refers to a feature under development that usually evolves into becoming a stable feature.
+  We encourage you to try out these features and report [bugs](https://github.com/localstack/localstack/issues/new/choose), missing functionality, or general feedback.
+* **Experimental** refers to a feature prototype that demonstrates initial feasibility and usually comes with many limitations.
+  Please share your feedback to shape such features and express your interest to get them prioritized.
+
+Disclaimer: Features under development (i.e., labelled as preview/beta, experimental/alpha, or similar terms) are subject to changes (e.g., behavior, pricing tiers) or even complete removal.
 
 ## Official Releases
 
+LocalStack follows [Semantic Versioning](https://semver.org/) except for [AWS parity](https://localstack.cloud/blog/2022-08-04-parity-explained) fixes,
```

Comment:
@alexrashed I wanted to point out this change for you, so we are aligned with communicating releases/changelogs properly 👀

Reply:
Yeah, looks good to me! We haven't really officially committed to it publicly, but our users expect it and we act on it anyways ;) So 👍🏽 from my side.